src/cloel.clj

```diff
@@ -1,11 +1,10 @@
 (ns cloel
   (:require [clojure.edn :as edn])
-  (:import [java.net ServerSocket Socket]
-           [java.io BufferedReader InputStreamReader PrintWriter]
-           [java.util.concurrent ConcurrentHashMap]))
```

Comment:
bb does not support `java.util.concurrent.ConcurrentHashMap`
I guess this has better performance over `atom`?

Reply:
We chang `call-results` to atom ? 

call-results just hold function callback result, I think atom is enough.